digital-workspace/latest/develop/developadfapp.md

```diff
@@ -0,0 +1,108 @@
+---
+title: Developing ADF based applications
+---
+
+In this section you are going to learn how to develop a custom ADF based application. Once made you can can start adding features, and customizing it. The creation of this basic ADF-based application is possible through the scaffolding tool called Yeoman. This means in a small amount of time you can create a working ADF based application in your development environment.
+
+<This should go somewhere in the beginning> You may consider adopting an ADF-based application in case you want to benefit from the ADF services and visual components provided by the framework. The Alfresco Yeoman Generator used for this purpose is available for free as an Open Source project on GitHub.
+
+Below you can find the available tutorials on the most common and requested tasks about developing ADF based applications.
+
+## Create an ADF based application using the Yeoman Generator
+
+This tutorial explains how to create a simple ADF-based application that you can start enriching with features, behaviors and customizations. The creation of this basic ADF-based application is possible through a scaffolding tool called Yeoman, which makes this task very straightforward.
+
+More in particular check that you have:
+
+Alfresco Content Services (alias ACS) Enterprise edition up and running (identify the URL that will be required as configuration). 
+
+The latest lts version of NodeJs.
+
+A recent (and supported) version of a browser (see here for further details).
+
+All the Angular development is done using the Typescript language. With this in mind, it is highly suggested to adopt a good text editor to help you in this task. We recommend Visual Studio Code a free, lightweight, and very powerful tool from Microsoft that works well with Angular development and has a big ecosystem of plugins to make the developer experience even better.
```

Comment:
I recommend turning "Visual Studio Code" block into a hyperlink with https://code.visualstudio.com/ address to help readers navigate fast.

Reply:
Removed Microsoft VS Code reference as per another person's suggestion.